clinical-clusters/2018/clinical-clusters.json

```diff
@@ -219,17 +219,6 @@
           "226"
         ]
       }
-    ],
-    "clinicalClusters": [
-      {
-        "name": "generalCare",
-        "measureIds": [
-          "130",
-          "226",
```

Comment:
does this measureId (`226`) need to be removed from the other clinicalClusters.measureIds definitions to avoid confusion?

Reply:
No, its kinda of a weird setup, here is the requirement for the "generalCare" cluster: "When submitting #130 and/or #226, they are not subject to EMA for these clinically related measures. It is expected to submit #130, #226, if quality measures if #134 and/or #317 is submitted." so the proper "interpretation" of this would be if 134 and/or 317 is submitted they have to submit 130 and 226 to make up the cluster. However they can also only submit 130 and or 226 by themselves which will then go down the regular path or "non-cluster" path. Hope that makes sense...